subsystem-arm/implementations/arm-modes.hpp

```diff
@@ -9,10 +9,30 @@ namespace sjsus::arm
         public:
 
         static arm_arguments SimulataneousMode(arm_commands commands)
-        {}
+        {
+            arm_arguments temp;
+            temp.rotunda_angle = anglesconvert(commands.rotunda_angle);
+            temp.shoulder_angle = anglesconvert(commands.shoulder_angle);
+            temp.elbow_angle = anglesconvert(commands.elbow_angle);
+            temp.wrist_pitch_angle = anglesconvert(commands.wrist_pitch_angle);
+            temp.wrist_yaw_angle = anglesconvert(commands.wrist_yaw_angle);
+            temp.finger_angle = anglesconvert(commands.finger_angle);
+            return{}
```

Comment:
make sure you return temp
```suggestion
        {
            arm_arguments temp;
            temp.rotunda_angle = anglesconvert(commands.rotunda_angle);
            temp.shoulder_angle = anglesconvert(commands.shoulder_angle);
            temp.elbow_angle = anglesconvert(commands.elbow_angle);
            temp.wrist_pitch_angle = anglesconvert(commands.wrist_pitch_angle);
            temp.wrist_yaw_angle = anglesconvert(commands.wrist_yaw_angle);
            temp.finger_angle = anglesconvert(commands.finger_angle);
            return temp;
```